pkg/webui/console/components/gateway-data-form/index.js

```diff
@@ -138,6 +142,12 @@ class GatewayDataForm extends React.Component {
           label={sharedMessages.public}
           description={m.locationDescription}
         />
+        <Form.Field
```

Comment:
the `title` is missing.
<img width="723" alt="Screenshot 2020-03-22 at 10 41 03" src="https://user-images.githubusercontent.com/16374166/77245758-a8d1c480-6c29-11ea-913e-21c644c28063.png">

Reply:
```
libs.bundle.js:119821 Warning: Failed prop type: The prop `title` is marked as required in `FormField`, but its value is `undefined`.
    in FormField (created by GatewayDataForm)
    in GatewayDataForm (created by GatewayAdd)
	...
``

Reply:
Title was left empty on purpose (also see PR description), but added it since it is required.

Reply:
I agree that in cases like that, it's fine to leave the title prop empty. We should look into ways to achieve that without sacrificing the title prop requirement, which is helpful in 99% of other cases.
For now, can you change the title to `Location Updates`?